packages/backend/src/managers/playground.ts

```diff
@@ -217,6 +218,7 @@ export class PlayGroundManager {
       try {
         const response = await fetch(`http://localhost:${freePort}`);
         contacted = response.ok;
```

Comment:
On my side (mac), I get a response with value `{}`. 
I would set `contacted = true` here, as the socket connection errors we want to detect go to the catch, so if we are at this point, socket connection is ok

Reply:
Should be fixed by 528e991